src/main/java/com/solvdLaba/airport/Airline.java

```diff
@@ -1,16 +1,39 @@
 package com.solvdLaba.airport;
 
-import java.util.List;
-
-public interface Airline {
+import com.solvdLaba.exceptions.FlightNotFoundException;
 
-    void addFlight(Flight flight);
-
-    void removeFlight(Flight flight);
+import java.util.ArrayList;
+import java.util.List;
 
-    List<Flight> getAllFlights();
+public class Airline {
 
-    String getName();
+   private String name;
+   private List<Flight> flights;
 
-    void setName(String name);
+   public Airline(String name){
+       this.name = name;
+       this.flights = new ArrayList<>();
+   }
+    public int hashCode() {
```

Comment:
You `}` on line 16 and `public` are not on the same level. There is an extra space between them, which is not correct. Format code.

Reply:
Done